### PR TITLE
Crow Rising

### PR DESCRIPTION
**Pull request by CrowRising**
I give up, I don't understand why test taking completely derails me. I erased and re-cloned three times...I am so scrambled. I do not struggle this much during my practices. I need to figure out to set myself up for success in testing.

### CODE REVIEW

File: db/migrate/20230410172217_add_comments_to_customers.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class AddCommentsToCustomers < ActiveRecord::Migration[5.2]
+  def change
+    add_reference :customers, :item, foreign_key: true
+    add_reference :customers, :supermarket, foreign_key: true
```
Comment (jamisonordway) [inline db/migrate/20230410172217_add_comments_to_customers.rb:4]: We don't need these foreign keys on the `customers` table. Customers don't belong to an Item or belong to a Supermarket.

File: spec/models/customer_spec.rb [hunk 2]
```diff
@@ -0,0 +1,6 @@
+require 'rails_helper'
+
+RSpec.describe Customer, type: :model do
+  it { should have_many :cutomer_items }
```
Comment (jamisonordway) [inline spec/models/customer_spec.rb:4]: I wonder if you ran into any test failures due to the misspelling (`:cutomer_items` should be `:customer_items`)

File: db/migrate/20230410170931_creat_customer_item.rb [hunk 3]
```diff
@@ -0,0 +1,10 @@
+class CreatCustomerItem < ActiveRecord::Migration[5.2]
+  def change
+    create_table :customer_item do |t|
```
[the remark this answers is not in the document]
Comment (CrowRising) [reply db/migrate/20230410170931_creat_customer_item.rb:3]: This whole IC was a complete disaster, it was a fast and steep downward spiral and nothing that I turned is even worth commenting on tbh. This was the third iteration of the repo for me and my first was actually on the right path, this is very far from being anything remotely close to acceptable.

File: spec/models/customer_spec.rb (hunk 2 above)
Comment (jamisonordway) [reply spec/models/customer_spec.rb:4]: I'm sorry to hear it was such a frustrating experience for you, and I'm glad you turned this in anyway. Thanks for doing that.

**Pull request closed without merging**